File: problem_05/Problem_05.java
public class Problem_05 {
    public static void main(String[] args) {
        int num = 0;
        do {
            num++;
            if (num % 11 == 0 && num % 13 == 0 && num % 14 == 0 && num % 16 == 0 && num % 17 == 0 && num % 18 == 0 && num % 19 == 0 && num % 20 == 0) {
                break;
            }
        } while (true);

        System.out.println("The smallest positive number that is evenly divisible by all of the numbers from 1 to 20 is " + num);
    }
}

File: problem_05/problem_05.js
let number = 0;

do {
  number += 1;
  if (number % 11 === 0 && number % 13 === 0 && number % 14 === 0 && number % 16 === 0 && number % 17 === 0 && number % 18 === 0 && number % 19 === 0 && number % 20 === 0) {
    break;
  }
} while (true);

console.log("The smallest number that can be divided by each of the numbers from 1 to 20 without any remainder is: ", number);